README.md

```diff
@@ -37,6 +37,7 @@ Here is a quick example of how to install this action in any workflow:
 | `mode` | `false` | `"fail"` | The mode to run the action in `"warn"` or `"fail"` |
 | `comment` | `false` | `"false"` | Whether or not to comment on a PR with the validation results - `"true"` or `"false"` |
 | `base_dir` | `false` | `"."` | The base directory to search for JSON and YAML files (e.g. ./src) - Default is `"."` which searches the entire repository |
+| `files` | `false` | `""` | List of file paths to validate. Each file path must be on a newline. |
```

Comment:
Would both of these examples work?

**Example 1**:

```yaml
- name: json-yaml-validate
  id: json-yaml-validate
  uses: GrantBirki/json-yaml-validate@vX.X.X
  with:
    files: "test1.json\ntest2.json\ntest3.json"
```

**Example 2**:

```yaml
- name: json-yaml-validate
  id: json-yaml-validate
  uses: GrantBirki/json-yaml-validate@vX.X.X
  with:
    files: |
      test1.json
      test2.json
      test3.json
```

Reply:
Mainly just curious, thanks!

Reply:
I don't think the first will work. What's the best way to check?

Reply:
I'll merge and we can give it a go! If only the second option works, that totally fine. More curiosity than anything else.